.github/actions/setup-server/action.yml

```diff
@@ -0,0 +1,73 @@
+name: Setup Server
+description: "Sets up a server and installs the appropriate environment."
+
```

Comment:
This is copied from the aqueduct repo? What's the reason that we can't just use that one and can you document that clearly here?

Reply:
It's mostly the same as the aqueduct repo except the paths need to be modified / I need to cd in `../aqueduct` for many lines since it is not the base directory.
For example:
```
    - name: Update aqueduct with latest code
      shell: bash
      run: cd ../aqueduct; python3 scripts/install_local.py -g -s -e
```

Reply:
Got it. I'd prefer if we could use the same code. Can you just make a task to deduplicate the two? Note that it would involve passing in the base directory as an argument into `setup-server.yml`.

Reply:
Gotcha. Here is the task: https://linear.app/aqueducthq/issue/ENG-2800/de-duplicate-aqueduct-and-gitbook-github-actions